src/subplots.ts

```diff
@@ -2287,10 +2288,12 @@ export class newScatter extends Frame {
     public is_in_multiplot: boolean = false
     ) {
       super(data, width, height, buttons_ON, X, Y, canvas_id, is_in_multiplot);
-      if (!data.tooltip) this.tooltipAttributes = Array.from(this.features.keys());
-      else this.tooltipAttributes = data.tooltip.attribute;
-      this.unpackPointStyle(data);
-      this.computePoints();
+      if (this.nSamples > 0) {
+        if (!data.tooltip) this.tooltipAttributes = Array.from(this.features.keys());
```

Comment:
Could use ternary operator if you fancy it

Reply:
Thanks